src/spikegadgets_to_nwb/convert_ephys.py

```diff
@@ -72,11 +72,25 @@ def __init__(
             self.nwb_hw_channel_order = nwb_hw_channel_order
 
         # NOTE: this will read all the timestamps from the rec file, which can be slow
-        self.timestamps = []
-        [
-            self.timestamps.extend(neo_io.get_analogsignal_timestamps(0, n_time))
-            for neo_io, n_time in zip(self.neo_io, self.n_time)
-        ]
+        if self.neo_io[0].sysClock_byte:  # use this if have sysClock from ptp
```

Comment:
Little confused by this because PTP should not factor into anything but the position camera(s) timing.